src/lib/provable/bool.ts

```diff
@@ -258,6 +257,40 @@ class Bool {
     return new Bool(x).or(y);
   }
 
+  /**
+   * Boolean AND operation across a list of booleans, returns `Bool(true)` if all elements in the list are true.
+   */
+  static allTrue(list: (Bool | boolean)[]): Bool {
```

Comment:
Is this included from a different PR?  If not, should it be separated?

Reply:
cherry picked from https://github.com/o1-labs/o1js/pull/2038